### PR TITLE
AUTO-183: Updates

### PR DESCRIPTION
**Pull request by jh-bate**
- consistent use of properties e.g. `screen.MyButton` rather than `screen.MyButton()`
- actions occur on properties e.g. `screen.MyButton.tap()`
- eslint setup
- remove generic `ScrollToBottom` or `ScrollToTop` and instead use more specific `SwipeUpUntilVisible(labelToSee)` or `SwipeDownUntilVisible(labelToSee)` that works across device sizes
- rationalise tests

### CODE REVIEW

File: e2e/error_1/error_1.spec.js [hunk 1]
```diff
@@ -1,30 +1,34 @@
...
-                history: { name: cgmText.History.BackfillGlucose, backfillHours: 6 },
-            }
+            cgmData: cgmData
         });
         await test.prepare();
     });
-    describe('from pump', () => {
-        pumpTests(test);
+    describe('from cgm', () => {
+        cgmTests(test, cgmData);
```
Comment (darinkrauss) [inline e2e/error_1/error_1.spec.js]: [nit] Doesn't the test already have the `cgmData`?

File: src/base/entryScreen.js [hunk 2]
```diff
@@ -6,27 +6,24 @@ class EntryScreen extends Screen {
...
         return match.accessible.Button(this.generalText.ButtonLabel.InfoCircle);
     }
-    GuardrailWarningIconPicker() {
+    get GuardrailWarningIconPicker() {
         return match.accessible.Image(this.generalText.Alert.ExclamationMark).atIndex(0);
     }
+    get CancelNewEntryButton() {
+        return match.accessible.Button(this.generalText.Cancel);
+    }
     GuardrailMessage(text) {
```
Comment (darinkrauss) [inline src/base/entryScreen.js:21]: [nit] `get`?

**Review by darinkrauss — APPROVED** [review]
LGTM! 🚀

File: src/base/entryScreen.js (hunk 2 above)
Comment (jh-bate) [reply src/base/entryScreen.js:21]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get

File: e2e/error_1/error_1.spec.js (hunk 1 above)
Comment (jh-bate) [reply e2e/error_1/error_1.spec.js]: 🤦 yup ... have updated thanks

**Pull request merged**